recipe/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
   run:
     - python
     - pydantic >=1,<3
-    - langsmith >=0.1.0,<0.2.0
-    - tenacity >=8.1.0,<9.0.0
+    - langsmith >=0.1.75,<0.2.0
+    - tenacity >=8.1.0,!=8.4.0,<9.0.0
     - jsonpatch >=1.33.0,<2.0.0
     - pyyaml >=5.3
     - packaging >=23.2.0,<24.0.0
```

Comment:
Am I looking at the correct package and a branch https://github.com/langchain-ai/langchain/blob/langchain-core%3D%3D0.3.12/libs/core/pyproject.toml#L36-L41? Because there are different pinnings.

Reply:
Right, I forgot to change them when I bumped the version.